feat(RTE): add custom column break plugin

fulopdaniel opened the pull request:
Introducing container query responsivity to the rich text editor when it comes to columns.

1. Extending fondue ColumnBreak plugin with some custom logic, so guideline-blocks can stay untouched
2. Extending serialized text with the same logic

fulopdaniel commented on packages/guideline-blocks-settings/src/components/Attachments/Attachments.spec.ct.tsx:
```diff
@@ -103,7 +103,6 @@ describe('Attachments', () => {
         });
 
         if ((await isPre302Stub(appBridge)) && hasOpenAssetChooser(appBridge)) {
-            // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```
`pnpm lint:fix` fixed this one

ragi96 replied:
Probably not, pretty sure tests will fail with AppBridge 3.0.0

fulopdaniel replied:
yepppp...

ragi96 reviewed (commented):
Don't we need to replace the `getColumnBreakClasses` in the other Plugins?

And also add tests for the behaviour you add here, so we are sure it works as expected 💯

ragi96 commented on .changeset/spotty-papayas-admire.md:
```diff
@@ -0,0 +1,5 @@
+"@frontify/guideline-blocks-settings": patch
+---
+
+feat(RTE): add custom column break plugin
```
Maybe add a bit of stuff here about how the new Plugin can be used

fulopdaniel commented:
> Don't we need to replace the `getColumnBreakClasses` in the other Plugins?
> 
> And also add tests for the behaviour you add here, so we are sure it works as expected 💯

`getColumnBreakClasses` does not need to be replaced, it is for the dashed underline. As I am using the plugin from Fondue and just added some functionality on top, it continues to work as expected.

Added the tests.

ragi96 commented on .changeset/spotty-papayas-admire.md:
```diff
@@ -0,0 +1,6 @@
+---
+"@frontify/guideline-blocks-settings": patch
```
```suggestion
"@frontify/guideline-blocks-settings": minor
```

ragi96 reviewed (commented):
Just one last thing
